ui/src/publication/card/publication-list.tsx

```diff
@@ -8,6 +8,11 @@ type PublicationListProps = {
     publications: PublicationDetails[];
 };
 
+const HIDDEN_PUBLICATION_CAUSES = [
+    PublicationCause.LAYOUT_DESIGN_CANCELLATION,
+    PublicationCause.CALCULATED_CHANGE,
+];
```

Comment:
Nämä piilotetaan käyttäjiltä, sillä:
- `LAYOUT_DESIGN_CANCELLATION`-julkaisut luotuvat aina samaan aikaan `LAYOUT_DESIGN_DELETE`-julkaisujen kanssa (julkaisuja syntyy kaksi lähinnä teknisistä syistä ja kumpaakaan ei voi syntyä ilman toista). Varsinainen suunnitelman poisto myös tapahtuu tuossa `LAYOUT_DESIGN_DELETE`-julkaisussa -> piilotetaan nämä
- Pähkäiltiin Karin kanssa että `CALCULATED_CHANGE`-julkaisuilla ei oikeastaan ole kauheasti lisäarvoa kenellekään käyttäjälle. Näitähän syntyy siis suunnitelmajulkaisuihin esim. silloin, kun main-officialiin julkaistaan jotain sellaista mikä vaikuttaa suunnitelmaankin (esim. ka. suunnitelmasta mainiin vietyjen assettien julkaisu)